Zend/tests/magic_methods_set_state.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Testing __set_state() declaration with wrong modifier
+--FILE--
+<?php
+
+class a {
+    function __set_state()
```

Comment:
Looks like `__set_state` does have the hability of not take any arguments and return something different from array:  https://3v4l.org/JptYd.

I've opened [a bug](https://bugs.php.net/bug.php?id=79521) to fix that in the [Magic Method's documentation](https://www.php.net/manual/en/language.oop5.magic.php#object.set-state).

Reply:
Actually, that's a bug, the same one we fixed in #5441.